include/unifex/stop_when.hpp

```diff
@@ -92,18 +92,16 @@ namespace unifex
         return r.get_stop_token();
       }
 
-      template(typename CPO, typename... Args)
+      template(typename CPO)
           (requires is_receiver_query_cpo_v<CPO>)
       friend auto tag_invoke(
```

Comment:
If I remember correctly, it was because the receiver query `tag_invoke` overloads were interfering with the `connect` `tag_invoke` overloads and causing spurious failures. The failures may have been msvc-specific, I can't quite recall.

Reply:
In this instance I was trying to eliminate an ambiguous call to `visit_continuations()` where it was picking up the default `tag_invoke(_fn, const Receiver&, Func&&)` implementation for `visit_continuations()` and the receiver's forwarding tag_invoke overload `tag_invoke(CPO, const type& r, Args&&...)`. It may not have been necessary after changing the default implementation to not be defined as a `tag_invoke` overload.

But, yes, I removed `Args...` from most query-forwarding overloads to avoid these overloads being picked up by `connect()`. Keeping the arity strictly less than the arity of `connect()` means that these are not considered which avoided some unnecessary deep instantiations and some weird template errors.